### PR TITLE
chore(update): Home Page Promos

### PR DESCRIPTION
**Pull request by dnilasor**
`In this PR, I've updated the conference ad block on the home page to feature information and links for Spinnaker Gardening Days. I've also added a simple Join Slack CTA under that conference ad block, with the slack logo. Ideally this block would be a bit narrower so that it doesn't push other page content down so much, but to me this is acceptable in the short-term while we refresh the site. My reasoning for adding the CTA is that, I'm worried folks are coming to the site home page now and not knowing where to turn for help. We can help OSS users much better on Slack, so they don't get stuck and frustrated on their own.
@brian-armory @dorbin @aimeeu take a peek if you like.

### CODE REVIEW

**Comment by 317brian** [comment]
LGTM. One question before I approve and merge, is the image public domain or some other free use license?

File: index.md
Comment (dorbin) [inline index.md]: @dnilasor do you mean to keep the previous summit-row name? No big deal if you don't change it.

**Review by dorbin — APPROVED** [review]
LGTM, with one small comment. 
Thanks!

File: index.md
Comment (dnilasor) [reply index.md]: I did because it seemed like a generic-enough row name. Maybe I should change it to an event more generic name (promo_row ?) so we don't need to change it? @dorbin maybe there is some value there

**Comment by dnilasor** [comment]
I added another change with a descriptive message. Please LMK if anything else needed before approval!

File: index.md
Comment (dorbin) [inline index.md]: Hm, might be better to link directly to join.spinnaker.io rather than to an Armory url that redirects there?

File: index.md
Comment (dorbin) [inline index.md]: Inadvertent?

File: index.md
Comment (dorbin) [inline index.md:3]: Yeah, this!

File: index.md
Comment (dorbin) [inline index.md]:
```suggestion
  excerpt: "Visit [join.spinnaker.io](https://go.armory.io/spinslack) to get help on Spinnaker Slack, a place for the community to come together. Use this vibrant workspace to ask and answer questions, connect with other operators and users, discuss issues with SIGs, and learn about Spinnaker!"
```

Comment (dorbin) [reply index.md]: Just because "..and learn Spinnaker" almost implies Spinnaker Slack is sufficient for learning Spinnaker.

File: index.md
Comment (dnilasor) [reply index.md]: Yes, wth oops

Comment (dnilasor) [reply index.md]: fixed

File: index.md
Comment (dnilasor) [reply index.md]: @dorbin understood, here's my commit message on this:

>  I received a request that we be able to track how many folks join slack using the link from spinnaker.io. The easiest way for me to do this currently is to use Armory's existing rebrandly account which includes detailed link tracking, so I've added that URL here. I'm sure there is a way to do this with Google Analytics and Tag Manager, although I don't know it off the top of my head. I want to know, are we comfortable with me using the rebrandly approach if we're not using it for anything other than tracking? Or do I need to look into an alternative approach? Thanks!

I'd love your thoughts @dorbin @brian-armory !

**Comment by dnilasor** [comment]
@plumpy, makes sense. Thanks for putting #1646 on my radar. Seems like we need to sort out implementation details, so for now I'll just re-add the simple join.spinnaker.io link so we can get this approved. Then I'll look for the most maintainable strategy. Thanks for looking @dorbin @plumpy !!

**Comment by dnilasor** [comment]
@brian-armory, changed the link to something not controversial and will open another PR that addresses this and #1646. Mergey merge time please?

**Review by dorbin — APPROVED** [review]

**Comment by 317brian** [comment]
Thanks for making sure the analytics discussion doesn't get lost @dnilasor. I'll go ahead and merge since Dave already approved.

**Comment by 317brian** [comment]
Weird. What appears on the OSS site is not what showed locally when I built it originally. @dnilasor will troubleshoot.

**Pull request merged**
